runtimes/rust/tests/migration.rs

```diff
@@ -342,7 +346,34 @@ fn resume_with_fast_forward_migration() {
         ),
     )
     .unwrap();
-    drop(events_handle.take());
+    assert_eq!(
+        events_handle.take(),
+        vec![
+            RuntimeEvent::BeforeTransactions(Height(4), ToySupervisorService::INSTANCE_ID),
```

Comment:
Nit: Could you please check that the resumed service receives `before` / `after_transactions` calls?